.studio/compliance-service

```diff
@@ -87,3 +87,12 @@ function compliance_configure_purge() {
     }
   }"
 }
+
+document "load_compliance_reports" <<DOC
+  Loads compliance report.
+DOC
+function load_compliance_reports() {
+    cd components/compliance-service/test_data/audit_reports
+    ./send_to_data_collector.sh https://a2-dev.test "$(get_admin_token)"
+    cd ../../../../
```

Comment:
`pushd` and `popd` are typically used in automate's shell scripts to avoid all the dots.

Reply:
ah interesting, ok ill make the change!